src/Microsoft.IdentityModel.Protocols/Configuration/HttpDocumentRetriever.cs

```diff
@@ -75,6 +76,18 @@ public HttpDocumentRetriever(HttpClient httpClient)
         /// </summary>
         public bool RequireHttps { get; set; } = true;
 
+        /// <summary>
+        /// If set specifies the protocol version to use when sending HTTP requests.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// If set, specifies the desired HTTP version to use for the requests.
        /// Otherwise, HttpClient.DefaultRequestVersion is used.
        /// </summary>
        /// <remarks>
        /// HttpRequestMessage.Version specifies the desired request version.
        /// HttpRequestMessage.VersionPolicy defines how strictly the HTTP version should be adhered to, allowing for fallback to other versions if necessary.
        /// </remarks>
```